le_utils/constants/content_kinds.py

```diff
@@ -24,6 +24,7 @@
     file_formats.MP4: VIDEO,
     file_formats.MP3: AUDIO,
     file_formats.PDF: DOCUMENT,
+    file_formats.EPUB: DOCUMENT,
```

Comment:
Need to figure out whether to make this a separate content kind (need to test this though). How does Studio handle two separate file formats for one content kind?

Reply:
Hopefully it does, because Kolibri does! And there will be more like this, so would rather maintain the extension kind separation.